demo/README.md

```diff
@@ -0,0 +1,28 @@
+# Demo
```

Comment:
I don't think we want / need this `README.md` as it is a standard one and doesn't add any value.